dipy/viz/regtools.py

```diff
@@ -4,6 +4,20 @@
 plt, _, _ = optional_package("matplotlib.pyplot")
 
 
+def _tile_plot(imgs, titles, **kwargs):
+    """
+    Helper function
+    """
+    # Create a new figure and plot the three images
+    fig, ax = plt.subplots(1, len(imgs))
+    for ii, a in enumerate(ax):
+        a.set_axis_off()
+        a.imshow(imgs[ii], **kwargs)
+        a.set_aspect('auto', 'box')
```

Comment:
OK - I have changed it. It makes sense for users to just use the returned
figure object to customize this as desired.

On Thu, Feb 5, 2015 at 2:34 PM, Omar Ocegueda <notifications@github.com>
wrote:

> In dipy/viz/regtools.py
> <https://github.com/nipy/dipy/pull/552#discussion_r24206520>:
>
> > @@ -4,6 +4,20 @@
> >  plt, _, _ = optional_package("matplotlib.pyplot")
> >
> >
> > +def _tile_plot(imgs, titles, **kwargs):
> > +    """
> > +    Helper function
> > +    """
> > +    # Create a new figure and plot the three images
> > +    fig, ax = plt.subplots(1, len(imgs))
> > +    for ii, a in enumerate(ax):
> > +        a.set_axis_off()
> > +        a.imshow(imgs[ii], **kwargs)
> > +        a.set_aspect('auto', 'box')
>
> People may not want to set auto-aspect ratio (e.g. when using
> "plot_slices" the coronal and sagital slices look weird), maybe add it as
> an optional parameter?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nipy/dipy/pull/552/files#r24206520>.
>